assets/dev/scss/editor/preview/preview.scss

```diff
@@ -904,12 +904,45 @@ html.elementor-html {
 }
 
 // Muted elements in the editor
-[data-elementor-device-mode="desktop"] .elementor-hidden-desktop,
-[data-elementor-device-mode="tablet"] .elementor-hidden-tablet,
-[data-elementor-device-mode="mobile"] .elementor-hidden-phone {
+
+[data-elementor-device-mode="desktop"] .elementor-section.elementor-hidden-desktop,
+[data-elementor-device-mode="tablet"] .elementor-section.elementor-hidden-tablet,
+[data-elementor-device-mode="mobile"] .elementor-section.elementor-hidden-phone {
+
+	> .elementor-element-overlay {
+		background-color: #ddd;
```

Comment:
Don't we need a color from our editor palette?

Reply:
Done